### ISSUE TITLE
Deleting 4 bar gas infrastructure

### ISSUE THREAD

**Issue by Sanne512** [quintel/etmoses#1169]
When you delete the gas infrastructure at 4 bar, which is a realistic situation, the load still displays a gas load at 4 bar. I tried this in the following LES: https://moses.energytransitionmodel.com/testing_grounds/506/gas_load

**Comment by antw** [comment]
If there is absolutely no infrastructure installed at a level in the gas network, it is assumed that capacity at that level is infinite, and efficiency is 100% in both directions (no loss). We only start limiting capacity and efficiency once something has been installed.

It would not be difficult to have capacity become zero in these situations; in which case no gas would be able to flow across a pressure level when there is no infrastructure there. @ChaelKruip What do you think?

**Comment by ChaelKruip** [comment]
> @ChaelKruip What do you think?

I think the current approach is logical. If nothing is installed it cannot lead to a constraint either (the absence of a pressure level implies that the surrounding levels are connected directly).

Closing this. If you disagree @Sanne512 please re-open this issue.

**Comment by Sanne512** [comment]
> (the absence of a pressure level implies that the surrounding levels are connected directly).

I agree with this part. And consequently for me it would make sense if you look at the yearly gas flow (see below) that the two columns of 0.125 bar - 4 bar and 4 bar- 8 bar would be merged to one column of 0.125 bar - 8 bar. I don't know if this is possible? 
![afstemming wensenlijst 1169](https://cloud.githubusercontent.com/assets/17741822/16873586/8fae8494-4a94-11e6-94c2-492a49366a71.PNG)

**Comment by ChaelKruip** [comment]
> I don't know if this is possible? 

Yes this is certainly possible but I'm not sure if it worth the effort. I'd have to talk to @antw and @grdw about this but my early estimate is that merging the bars based on the gas-assets list would take at least 5 man-days of work. 

Changing the label from 'bug' to 'feature'.

**Comment by Sanne512** [comment]
@ChaelKruip have you checked with @antw and @grdw how much effort this takes? 

Including @MarjoleinSchwachofer for further reference from Alliander

**Comment by ChaelKruip** [comment]
> @ChaelKruip have you checked with @antw and @grdw how much effort this takes?

About two man-days work. The chart has hard-coded data series and names. To make this dynamic is not incredibly difficult but needs quite a bit of code to be rewritten.
